tests/unit/models/blueprint-test.js

```diff
@@ -365,20 +364,23 @@ describe('Blueprint', function() {
 
         expect(output).to.equal(testString);
 
-        td.verify(blueprint._printCommand(), {ignoreExtraArgs: true, times: 0});
+        td.verify(blueprint._printCommand(), {times: 0});
       });
 
       it('calls printCommand', function() {
```

Comment:
Should we rename this test now?  It is no longer asserting that `printCommand` is called.